reark/src/main/java/io/reark/reark/data/stores/StoreGetInterface.java

```diff
@@ -0,0 +1,10 @@
+package io.reark.reark.data.stores;
+
+import rx.Observable;
+
+/**
+ * Created by ttuo on 27/06/16.
+ */
+public interface StoreGetInterface<T, U> {
+    Observable<U> getStream(T id);
```

Comment:
What comes to behaviour of getStream, I think renaming it might be the best way here. The logic that returns the last value could be done in many different ways, so I would rather keep it in the Store layer instead of StoreCore. Unless there would be a "universal" best way of always returning the last value without the limitation of startWith for new subscribers (values might have come between getStream and subscribe). A fail-safe getStream should be possible to do with a custom observable that gives new subscribers the last value automatically, though then this code would need to be copied in all different cores.